src/ex4/dist/clients/item_client.js

```diff
@@ -0,0 +1,35 @@
+class itemClient {
+    
+    constructor(){
+        this.todoList = [];
+    }
+
+    async getTodo() {
+       const response = await fetch("/getAll");
+       return await response.json();
+    }
+
+    async addTodo(todoList) {
+        const response=await fetch("/",{
+            method: 'POST',
+            headers: {'Content-Type': 'application/json'},
+            body: JSON.stringify(todoList)
+        })
+       return await response.json();
+     }
+
+     async deleteTodo(id) {
+        const response=await fetch("/"+id,{
```

Comment:
we usually format the string like this
when the URL will get more complicated you'll see its much clearer this way
```suggestion
        const response=await fetch(`/${id}`,{
```